src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,95 @@
+import React, { Children, Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import TabContent from './TabContent';
+import TabItem from './TabItem';
+import TabMenu from './TabMenu';
+
+class Tabs extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        const { activeTabId } = props;
+
+        this.state = {
+            activeTabId,
+            derivedProp: activeTabId // eslint-disable-line react/no-unused-state
```

Comment:
`derivedProp` sounds too abstract. Can we rename it to something like `derivedTabId` or store whole props object  as `prevProps`?

Reply:
Makes sense, I will change it to `derivedTabId` as I don't want to duplicate anything else that we don't need

Reply:
Updated